defaults/manager_persistent_snapshots.yaml

```diff
@@ -0,0 +1,18 @@
+aws:
+  snapshots:
+    10:
+      snapshot_tag: 'sm_20230223105105UTC'
+      expected_timeout: 1800  # 30 minutes
+      keyspace_name: "10gb_sizetiered"
+      confirmation_stress_command: "cassandra-stress read cl=QUORUM n=10485760 -schema 'keyspace=10gb_sizetiered replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=1..10485760"
+    100:
+      snapshot_tag: 'sm_20230223130733UTC'
+      expected_timeout: 9000  # 150 minutes
+      keyspace_name: "100gb_sizetiered"
+      confirmation_stress_command: "cassandra-stress read cl=QUORUM n=104857600 -schema 'keyspace=100gb_sizetiered replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=1..104857600"
+    2048:
+      snapshot_tag: 'sm_20230226074656UTC'
```

Comment:
In a followup PR:
Add another backup of 1TB that is taken with ICS.
Taken with RF=3 and a NetworkTopology in the schema.
When writing the dataset, use only one loader (you can ask @fgelcer for commands).

Then, add a test that restores this 1TB into a 9 nodes cluster (smaller ones - e.g. i4i.xlarge).
When restoring the data, measure the time and report it to the test result of how long it took to restore.